### ISSUE TITLE
2025-06-10, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#196]
# Today's Arxiv Papers

(1) [Play to Generalize: Learning to Reason Through Game Play](http://arxiv.org/abs/2506.08011v1), 2025-06-09

(2) [GUI-Reflection: Empowering Multimodal GUI Models with Self-Reflection Behavior](http://arxiv.org/abs/2506.08012v1), 2025-06-09

(3) [Hidden in plain sight: VLMs overlook their visual representations](http://arxiv.org/abs/2506.08008v1), 2025-06-09

(4) [Reinforcement Pre-Training](http://arxiv.org/abs/2506.08007v1), 2025-06-09

(5) [Aligning Text, Images, and 3D Structure Token-by-Token](http://arxiv.org/abs/2506.08002v1), 2025-06-09

(6) [Reparameterized LLM Training via Orthogonal Equivalence Transformation](http://arxiv.org/abs/2506.08001v1), 2025-06-09

(7) [Supporting Construction Worker Well-Being with a Multi-Agent Conversational AI System](http://arxiv.org/abs/2506.07997v1), 2025-06-09

(8) [HeuriGym: An Agentic Benchmark for LLM-Crafted Heuristics in Combinatorial Optimization](http://arxiv.org/abs/2506.07972v1), 2025-06-09

(9) [CyberV: Cybernetics for Test-time Scaling in Video Understanding](http://arxiv.org/abs/2506.07971v1), 2025-06-09

(10) [SpaCE-10: A Comprehensive Benchmark for Multimodal Large Language Models in Compositional Spatial Intelligence](http://arxiv.org/abs/2506.07966v1), 2025-06-09

(11) [SlideCoder: Layout-aware RAG-enhanced Hierarchical Slide Generation from Design](http://arxiv.org/abs/2506.07964v1), 2025-06-09

(12) [Reinforcing Multimodal Understanding and Generation with Dual Self-rewards](http://arxiv.org/abs/2506.07963v1), 2025-06-09

(13) [Correlated Errors in Large Language Models](http://arxiv.org/abs/2506.07962v1), 2025-06-09

(14) [TokenBreak: Bypassing Text Classification Models Through Token Manipulation](http://arxiv.org/abs/2506.07948v1), 2025-06-09

(15) [Statistical Hypothesis Testing for Auditing Robustness in Language Models](http://arxiv.org/abs/2506.07947v1), 2025-06-09

(16) [ProtocolLLM: RTL Benchmark for SystemVerilog Generation of Communication Protocols](http://arxiv.org/abs/2506.07945v1), 2025-06-09

(17) [Decoupling the Image Perception and Multimodal Reasoning for Reasoning Segmentation with Digital Twin Representations](http://arxiv.org/abs/2506.07943v1), 2025-06-09

(18) [Adversarial Attack Classification and Robustness Testing for Large Language Models for Code](http://arxiv.org/abs/2506.07942v1), 2025-06-09

(19) [Solving Inequality Proofs with Large Language Models](http://arxiv.org/abs/2506.07927v1), 2025-06-09

(20) [LUCIFER: Language Understanding and Context-Infused Framework for Exploration and Behavior Refinement](http://arxiv.org/abs/2506.07915v1), 2025-06-09

(21) [WeThink: Toward General-purpose Vision-Language Reasoning via Reinforcement Learning](http://arxiv.org/abs/2506.07905v1), 2025-06-09

(22) [MiniCPM4: Ultra-Efficient LLMs on End Devices](http://arxiv.org/abs/2506.07900v1), 2025-06-09

(23) [MEMOIR: Lifelong Model Editing with Minimal Overwrite and Informed Retention for LLMs](http://arxiv.org/abs/2506.07899v1), 2025-06-09

(24) [Evaluating Large Language Models on the Frame and Symbol Grounding Problems: A Zero-shot Benchmark](http://dx.doi.org/10.5281/zenodo.15617317), 2025-06-09

(25) [SoK: Data Reconstruction Attacks Against Machine Learning Models: Definition, Metrics, and Benchmark](http://arxiv.org/abs/2506.07888v1), 2025-06-09

(26) [Learning to Focus: Causal Attention Distillation via Gradient-Guided Token Pruning](http://arxiv.org/abs/2506.07851v1), 2025-06-09

(27) [SAM2Auto: Auto Annotation Using FLASH](http://arxiv.org/abs/2506.07850v1), 2025-06-09

(28) [HAIBU-ReMUD: Reasoning Multimodal Ultrasound Dataset and Model Bridging to General Specific Domains](http://arxiv.org/abs/2506.07837v1), 2025-06-09

(29) [Improving large language models with concept-aware fine-tuning](http://arxiv.org/abs/2506.07833v1), 2025-06-09

(30) [Addition in Four Movements: Mapping Layer-wise Information Trajectories in LLMs](http://arxiv.org/abs/2506.07824v1), 2025-06-09

(31) [Guideline Forest: Experience-Induced Multi-Guideline Reasoning with Stepwise Aggregation](http://arxiv.org/abs/2506.07820v1), 2025-06-09

(32) [WebUIBench: A Comprehensive Benchmark for Evaluating Multimodal Large Language Models in WebUI-to-Code](http://arxiv.org/abs/2506.07818v1), 2025-06-09

(33) [LLM Unlearning Should Be Form-Independent](http://arxiv.org/abs/2506.07795v1), 2025-06-09

(34) [Language-Vision Planner and Executor for Text-to-Visual Reasoning](http://arxiv.org/abs/2506.07778v1), 2025-06-09

(35) [REMoH: A Reflective Evolution of Multi-objective Heuristics approach via Large Language Models](http://arxiv.org/abs/2506.07759v1), 2025-06-09

(36) [Augmenting LLMs' Reasoning by Reinforcing Abstract Thinking](http://arxiv.org/abs/2506.07751v1), 2025-06-09

(37) [Research quality evaluation by AI in the era of Large Language Models: Advantages, disadvantages, and systemic effects](http://arxiv.org/abs/2506.07748v1), 2025-06-09

(38) [ArchiLense: A Framework for Quantitative Analysis of Architectural Styles Based on Vision Large Language Models](http://arxiv.org/abs/2506.07739v1), 2025-06-09

(39) [RSafe: Incentivizing proactive reasoning to build robust and adaptive LLM safeguards](http://arxiv.org/abs/2506.07736v1), 2025-06-09

(40) [Language Embedding Meets Dynamic Graph: A New Exploration for Neural Architecture Representation Learning](http://arxiv.org/abs/2506.07735v1), 2025-06-09

(41) [NeurIPS 2025 E2LM Competition : Early Training Evaluation of Language Models](http://arxiv.org/abs/2506.07731v1), 2025-06-09

(42) [Swiss Parliaments Corpus Re-Imagined (SPC_R): Enhanced Transcription with RAG-based Correction and Predicted BLEU](http://arxiv.org/abs/2506.07726v1), 2025-06-09

(43) [Interaction Analysis by Humans and AI: A Comparative Perspective](http://dx.doi.org/10.1145/3713043.3731527), 2025-06-09

(44) [Training Superior Sparse Autoencoders for Instruct Models](http://arxiv.org/abs/2506.07691v1), 2025-06-09

(45) [QUITE: A Query Rewrite System Beyond Rules with LLM Agents](http://arxiv.org/abs/2506.07675v1), 2025-06-09

(46) [How Benchmark Prediction from Fewer Data Misses the Mark](http://arxiv.org/abs/2506.07673v1), 2025-06-09

(47) [MCPWorld: A Unified Benchmarking Testbed for API, GUI, and Hybrid Computer Use Agents](http://arxiv.org/abs/2506.07672v1), 2025-06-09

(48) [GaRAGe: A Benchmark with Grounding Annotations for RAG Evaluation](http://arxiv.org/abs/2506.07671v1), 2025-06-09

(49) [Synthesis by Design: Controlled Data Generation via Structural Guidance](http://arxiv.org/abs/2506.07664v1), 2025-06-09

(50) [Beyond Benchmarks: A Novel Framework for Domain-Specific LLM Evaluation and Knowledge Mapping](http://arxiv.org/abs/2506.07658v1), 2025-06-09

(51) [Foundation Model Empowered Synesthesia of Machines (SoM): AI-native Intelligent Multi-Modal Sensing-Communication Integration](http://arxiv.org/abs/2506.07647v1), 2025-06-09

(52) [Evaluating LLMs Robustness in Less Resourced Languages with Proxy Models](http://arxiv.org/abs/2506.07645v1), 2025-06-09

(53) [TreeReview: A Dynamic Tree of Questions Framework for Deep and Efficient LLM-based Scientific Peer Review](http://arxiv.org/abs/2506.07642v1), 2025-06-09

(54) [SWE-Dev: Building Software Engineering Agents with Training and Inference Scaling](http://arxiv.org/abs/2506.07636v1), 2025-06-09

(55) [Unblocking Fine-Grained Evaluation of Detailed Captions: An Explaining AutoRater and Critic-and-Revise Pipeline](http://arxiv.org/abs/2506.07631v1), 2025-06-09

(56) [Event-Priori-Based Vision-Language Model for Efficient Visual Understanding](http://arxiv.org/abs/2506.07627v1), 2025-06-09

(57) [Intent Matters: Enhancing AI Tutoring with Fine-Grained Pedagogical Intent Annotation](http://arxiv.org/abs/2506.07626v1), 2025-06-09

(58) [LoRMA: Low-Rank Multiplicative Adaptation for LLMs](http://arxiv.org/abs/2506.07621v1), 2025-06-09

(59) [Vuyko Mistral: Adapting LLMs for Low-Resource Dialectal Translation](http://arxiv.org/abs/2506.07617v1), 2025-06-09

(60) [PolitiSky24: U.S. Political Bluesky Dataset with User Stance Labels](http://arxiv.org/abs/2506.07606v1), 2025-06-09

(61) [SceneRAG: Scene-level Retrieval-Augmented Generation for Video Understanding](http://arxiv.org/abs/2506.07600v1), 2025-06-09

(62) [Instructing Large Language Models for Low-Resource Languages: A Systematic Study for Basque](http://arxiv.org/abs/2506.07597v1), 2025-06-09

(63) [TwinBreak: Jailbreaking LLM Security Alignments based on Twin Prompts](http://arxiv.org/abs/2506.07596v1), 2025-06-09

(64) [Evaluating LLMs Effectiveness in Detecting and Correcting Test Smells: An Empirical Study](http://arxiv.org/abs/2506.07594v1), 2025-06-09

(65) [PrunePEFT: Iterative Hybrid Pruning for Parameter-Efficient Fine-tuning of LLMs](http://arxiv.org/abs/2506.07587v1), 2025-06-09

(66) [MalGEN: A Generative Agent Framework for Modeling Malicious Software in Cybersecurity](http://arxiv.org/abs/2506.07586v1), 2025-06-09

(67) [Beyond the Sentence: A Survey on Context-Aware Machine Translation with Large Language Models](http://arxiv.org/abs/2506.07583v1), 2025-06-09

(68) [Uncertainty-o: One Model-agnostic Framework for Unveiling Uncertainty in Large Multimodal Models](http://arxiv.org/abs/2506.07575v1), 2025-06-09

(69) [LLM-driven Indoor Scene Layout Generation via Scaled Human-aligned Data Synthesis and Multi-Stage Preference Optimization](http://arxiv.org/abs/2506.07570v1), 2025-06-09

(70) [SAFEFLOW: A Principled Protocol for Trustworthy and Transactional Autonomous Agent Systems](http://arxiv.org/abs/2506.07564v1), 2025-06-09

(71) [SELT: Self-Evaluation Tree Search for LLMs with Task Decomposition](http://arxiv.org/abs/2506.07557v1), 2025-06-09

(72) [ChemAgent: Enhancing LLMs for Chemistry and Materials Science through Tree-Search Based Tool Learning](http://arxiv.org/abs/2506.07551v1), 2025-06-09

(73) [Bit-level BPE: Below the byte boundary](http://arxiv.org/abs/2506.07541v1), 2025-06-09

(74) [MoQAE: Mixed-Precision Quantization for Long-Context LLM Inference via Mixture of Quantization-Aware Experts](http://arxiv.org/abs/2506.07533v1), 2025-06-09

(75) [BitVLA: 1-bit Vision-Language-Action Models for Robotics Manipulation](http://arxiv.org/abs/2506.07530v1), 2025-06-09

(76) [Learning What Reinforcement Learning Can't: Interleaved Online Fine-Tuning for Hardest Questions](http://arxiv.org/abs/2506.07527v1), 2025-06-09

(77) [IntenTest: Stress Testing for Intent Integrity in API-Calling LLM Agents](http://arxiv.org/abs/2506.07524v1), 2025-06-09

(78) [Towards Large Language Models with Self-Consistent Natural Language Explanations](http://arxiv.org/abs/2506.07523v1), 2025-06-09

(79) [LeVo: High-Quality Song Generation with Multi-Preference Alignment](http://arxiv.org/abs/2506.07520v1), 2025-06-09

(80) [Taking Flight with Dialogue: Enabling Natural Language Control for PX4-based Drone Agent](http://arxiv.org/abs/2506.07509v1), 2025-06-09

(81) [Large Language Models for Multilingual Vulnerability Detection: How Far Are We?](http://arxiv.org/abs/2506.07503v1), 2025-06-09

(82) [Graph-of-Causal Evolution: Challenging Chain-of-Model for Reasoning](http://arxiv.org/abs/2506.07501v1), 2025-06-09

(83) [Explicit Preference Optimization: No Need for an Implicit Reward Model](http://arxiv.org/abs/2506.07492v1), 2025-06-09

(84) [SpatialLM: Training Large Language Models for Structured Indoor Modeling](http://arxiv.org/abs/2506.07491v1), 2025-06-09

(85) [A Framework for Creating Non-Regressive Test Cases via Branch Consistency Analysis Driven by Descriptions](http://arxiv.org/abs/2506.07486v1), 2025-06-09

(86) [A Hybrid GA LLM Framework for Structured Task Optimization](http://arxiv.org/abs/2506.07483v1), 2025-06-09

(87) [Improving Fairness of Large Language Models in Multi-document Summarization](http://arxiv.org/abs/2506.07479v1), 2025-06-09

(88) [DeepVideo-R1: Video Reinforcement Fine-Tuning via Difficulty-aware Regressive GRPO](http://arxiv.org/abs/2506.07464v1), 2025-06-09

(89) [CCI4.0: A Bilingual Pretraining Dataset for Enhancing Reasoning in Large Language Models](http://arxiv.org/abs/2506.07463v1), 2025-06-09

(90) [From Calibration to Collaboration: LLM Uncertainty Quantification Should Be More Human-Centered](http://arxiv.org/abs/2506.07461v1), 2025-06-09

(91) [KScope: A Framework for Characterizing the Knowledge Status of Language Models](http://arxiv.org/abs/2506.07458v1), 2025-06-09

(92) [Language-Grounded Hierarchical Planning and Execution with Multi-Robot 3D Scene Graphs](http://arxiv.org/abs/2506.07454v1), 2025-06-09

(93) [When Style Breaks Safety: Defending Language Models Against Superficial Style Alignment](http://arxiv.org/abs/2506.07452v1), 2025-06-09

(94) [Extending Epistemic Uncertainty Beyond Parameters Would Assist in Designing Reliable LLMs](http://arxiv.org/abs/2506.07448v1), 2025-06-09

(95) [LlamaRec-LKG-RAG: A Single-Pass, Learnable Knowledge Graph-RAG Framework for LLM-Based Ranking](http://arxiv.org/abs/2506.07449v1), 2025-06-09

(96) [Fact in Fragments: Deconstructing Complex Claims via LLM-based Atomic Fact Extraction and Verification](http://arxiv.org/abs/2506.07446v1), 2025-06-09

(97) [LG-ANNA-Embedding technical report](http://arxiv.org/abs/2506.07438v1), 2025-06-09

(98) [Prompt to Protection: A Comparative Study of Multimodal LLMs in Construction Hazard Recognition](http://arxiv.org/abs/2506.07436v1), 2025-06-09

(99) [Well Begun is Half Done: Low-resource Preference Alignment by Weak-to-Strong Decoding](http://arxiv.org/abs/2506.07434v1), 2025-06-09

(100) [Plug-in and Fine-tuning: Bridging the Gap between Small Language Models and Large Language Models](http://arxiv.org/abs/2506.07424v1), 2025-06-09
